src/components/LegoGallery/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import ThemedImage from '@theme/ThemedImage';
+import clsx from 'clsx';
+
+const STANDARDS = [
+  {
+    title: 'LSP0 - ERC725 Account',
+    link: './universal-profile/lsp0-erc725account',
```

Comment:
I may be a bit nitpicky here, but I suggest running all the images through [`ImageOptim`](https://imageoptim.com/mac) again. I just tried one of the 16 new files, and it saved 30% of the data without any difference (as these are rendered and have sharp edges)

Let's make the docs more lean and accessible for mobile internet.